sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialTests.cs

```diff
@@ -395,22 +486,20 @@ public async Task VerifyIMDSRequestWithPodIdentityEnvVarResourceIdMockAsync()
             string query = request.Uri.Query;
 
             Assert.That(query, Does.Contain("api-version=2018-02-01"));
-            Assert.That(query, Does.Contain($"resource={Uri.EscapeDataString(ScopeUtilities.ScopesToResource(MockScopes.Default))}"));
+            Assert.That(query, Does.Contain($"resource={ScopeUtilities.ScopesToResource(MockScopes.Default)}"));
             Assert.That(Uri.UnescapeDataString(query), Does.Contain($"{Constants.ManagedIdentityResourceId}={_expectedResourceId}"));
-            Assert.IsTrue(request.Headers.TryGetValue("Metadata", out string actMetadataValue));
-            Assert.AreEqual("true", actMetadataValue);
         }
 
-        [NonParallelizable]
-        [Test]
+        // [NonParallelizable]
+        // [Test]
```

Comment:
Intentional commenting out of `[Test]`?

Reply:
Yes - I meant to totally delete these two commented tests because they validate the older version string for IMDS, which the MSAL implementation doesn't use.